src/components/Auth/helper.js

```diff
@@ -0,0 +1,59 @@
+const jwt = require('jsonwebtoken');
```

Comment:
helper us good, but you already has service file, use service for business logic